docs/android/emulator/mac-with-windows-vm.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Connect to an Android emulator on a Mac from inside a Windows VM"
+description: "Learn how to connect to an Android emulator running on a Mac from inside a Windows virtual machine."
+ms.date: 01/11/2024
+---
+
+# Connect to an Android emulator on a Mac from inside a Windows virtual machine
```

Comment:
Is this specifically about connecting to an Android emulator on a Mac from inside a Windows VM that's running on the same Mac?